map/tiler.sh

```diff
@@ -0,0 +1,53 @@
+#! /bin/bash
+
```

Comment:
Please add description to `README.md` explaining what this script is and (briefly) how to use it.

Reply:
Thanks!